source/getting-started.rst

```diff
@@ -118,4 +118,16 @@ The following describes the individual points in the diagram:
 8. The :ref:`postauth-endpoint` is used to fetch the results of the
    authentication.
 9. Nominally a ``RReq`` is returned to the Requestor. Parameters are detailed
-   in the :ref:`postauth response <postauth-response>` section.
\ No newline at end of file
+   in the :ref:`postauth response <postauth-response>` section.
+
+Sandbox environment
+===================
+
+A sandbox environment is included to ease integration. It is
```

Comment:
Perhaps `s/ease/facilitate/`

Reply:
Nah. Perhaps "A sandbox environment is provided as a service for initial integration, continuous integration and live tests."

Reply:
> A sandbox environment is included as a service for initial and continuous
> integration, and for live tests. This is our implementation so discrepancies
> with ACS implementations are unavoidable.